apps/crn-frontend/src/network/teams/state.ts

```diff
@@ -165,3 +176,18 @@ export const useAuthorSuggestions = () => {
       filters: new Set(),
     }).then(({ items }) => items);
 };
+
+export const usePostTeamResearchOutput = () => {
+  const authorization = useRecoilValue(authorizationState);
+  const setRefreshListing = useRefreshResearchOutputListing();
+  const setResearchOutputItem = useSetResearchOutputItem();
+  return async (payload: ResearchOutputPostRequest) => {
+    const researchOutput = await createTeamResearchOutput(
+      payload,
+      authorization,
+    );
+    setResearchOutputItem(researchOutput);
+    setRefreshListing();
```

Comment:
I think this is broken. Shouldn't it increase the integer?

Reply:
That's what setRefreshListing  is doing. Although we could add that call to setResearchOutputItem?

Reply:
Done as below